test/integration/connection-monitoring-and-pooling/multiple_clients.test.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+
+import { assert as test, setupDatabase } from '../shared';
+
+describe('Multiple Clients', function () {
```

Comment:
Did you accidentally rename this file multiple_clients when you intended to name it multiple_databases?

Reply:
No, it's testing multiple clients within the same process, this test is from the days where the db was the primary object from the driver.